Library/Formula/cattle.rb

```diff
@@ -1,24 +1,29 @@
-require 'formula'
-
 class Cattle < Formula
-  homepage 'https://github.com/andreabolognani/cattle#readme'
-  url 'https://github.com/andreabolognani/cattle/archive/cattle-1.0.1.tar.gz'
-  sha1 '1c60cdf8dc6b2b27bd4dfb79adea905a22e033c2'
+  homepage "https://github.com/andreabolognani/cattle"
+  url "https://github.com/andreabolognani/cattle/archive/cattle-1.2.0.tar.gz"
+  sha1 "714514796de2183177db413aee16a196437bb4d7"
+  head "https://github.com/andreabolognani/cattle.git"
 
-  depends_on 'glib'
-  depends_on 'gobject-introspection'
-  depends_on 'gtk-doc' => :build
-  depends_on 'pkg-config' => :build
+  # https://github.com/andreabolognani/cattle/issues/2
+  fails_with :clang do
+    build 600
+    cause "Unexpected nil-return of values"
+  end
```

Comment:
Sure, happy to wait on upstream here. This new release is from last October, and as far as I can see nobody has submitted it to us till now, so I don't think there's a particularly huge hankering for the update right this second.